Form/Extension/DisableCSRFExtension.php

```diff
@@ -22,22 +22,18 @@
  */
 class DisableCSRFExtension extends AbstractTypeExtension
 {
-    private $securityContext;
+    private $authorizationChecker;
     private $role;
 
-    public function __construct(SecurityContextInterface $securityContext, $role)
+    public function __construct(AuthorizationChecker $authorizationChecker, $role)
     {
-        $this->securityContext = $securityContext;
+        $this->authorizationChecker = $authorizationChecker;
         $this->role = $role;
     }
 
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
```

Comment:
please keep the ``setDefaultOptions(OptionsResolverInterface $resolver)`` calling the new ``configureOptions(OptionsResolver $resolver)`` as the forward compatibility layer is only implemented in AbstractTypeExtension in 2.7+, not in older versions, so you have to implement it yourselves here to keep compatibility with older versions